find_line_in_image.py

```diff
@@ -0,0 +1,72 @@
+# the modules we need for finding the distance and angle to the line
+import cv2
+import numpy as np
+import sys
+
+
+def find_line_contour(image):
+    opened_threshold_image = __clean_image(image)
+    # finds the contours in the image after open was used on it
+    # searching only for external contours
+    _, contours, _ = cv2.findContours(
+        cleaned_image, cv2.RETR_EXTERNAL,

```

Comment:
cleaned_image is not defined in this scop.

The script fails to run

Reply:
cleaned image is created in the __clean_image function